tests/transformers/test_transformers_model_export.py

```diff
@@ -3608,3 +3609,33 @@ def test_save_and_load_pipeline_without_save_pretrained_false(
         if callable(input_example):
             input_example = input_example()
         mlflow.pyfunc.load_model(model_path).predict(input_example)
+
+
+def test_persist_pretrained_model(small_seq2seq_pipeline):
+    with mlflow.start_run():
+        model_info = mlflow.transformers.log_model(
+            transformers_model=small_seq2seq_pipeline,
+            artifact_path="model",
+            save_pretrained=False,
+            pip_requirements=["mlflow"],  # For speed up logging
```

Comment:
smart idea :) we should probably do this for more tests!

Reply:
haha yea we still need keep some to test the requirement inference logic, but it doesn't have to repeat that many times like now:)

Reply:
I think as part of refactoring of the test modules for this flavor we should explicitly create an 'integration test suite' that tests happy path of 4-5 pipelines / components declarations and no more. Everything else can be migrated to other suites where we're not creating full pipelines and are mostly just mocking the downloads / instantiations of the pipelines themselves. 
What are your thoughts on that?

Reply:
Totally agree, current tests feel too much for dev inner-loop. We should have more  strictly "unit" tests isolating each functionality and mocking integration points.